[2016-03-09 07:43:39] lumen.ERROR: exception 'ReflectionException' with message 'Class App\Http\Controllers\App\Http\Controllers\PlaceController does not exist' in C:\wamp\www\lumen\vendor\illuminate\container\Container.php:738
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(738): ReflectionClass->__construct('App\\Http\\Contro...')
#1 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(633): Illuminate\Container\Container->build('App\\Http\\Contro...', Array)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Application.php(205): Illuminate\Container\Container->make('App\\Http\\Contro...', Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(516): Laravel\Lumen\Application->make('App\\Http\\Contro...')
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#10 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#11 {main}  
[2016-03-09 09:24:12] lumen.ERROR: exception 'ErrorException' with message 'Use of undefined constant console - assumed 'console'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:31
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(31): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Use of undefine...', 'C:\\wamp\\www\\lum...', 31, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->getPlace(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'getPlace', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-09 09:24:46] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function App\Http\Controllers\alert()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:31
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 09:33:24] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:21
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 09:33:43] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:21
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 09:34:16] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:21
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 09:34:56] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:21
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 10:34:05] lumen.ERROR: exception 'ErrorException' with message 'Missing argument 1 for App\Http\Controllers\PlaceController::getPlace()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:30
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(30): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(2, 'Missing argumen...', 'C:\\wamp\\www\\lum...', 30, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->getPlace()
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'getPlace', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-09 11:16:01] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:33
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 11:19:08] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:33
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 11:27:00] lumen.ERROR: exception 'ErrorException' with message 'Undefined index: place' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:32
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(32): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined index...', 'C:\\wamp\\www\\lum...', 32, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->getPlace()
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'getPlace', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-09 11:27:34] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:33
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 12:22:59] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:33
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 12:23:50] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:33
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 12:24:14] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Http\Controllers\DB' not found' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:33
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-09 12:30:35] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Domlur' in 'field list'' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:333
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(333): PDO->prepare('select `Domlur`...')
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(662): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'select `Domlur`...', Array)
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('select `Domlur`...', Array, Object(Closure))
#3 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(342): Illuminate\Database\Connection->run('select `Domlur`...', Array, Object(Closure))
#4 C:\wamp\www\lumen\vendor\illuminate\database\Query\Builder.php(1461): Illuminate\Database\Connection->select('select `Domlur`...', Array, true)
#5 C:\wamp\www\lumen\vendor\illuminate\database\Query\Builder.php(1447): Illuminate\Database\Query\Builder->runSelect()
#6 C:\wamp\www\lumen\vendor\illuminate\database\Eloquent\Builder.php(569): Illuminate\Database\Query\Builder->get(Array)
#7 C:\wamp\www\lumen\vendor\illuminate\database\Eloquent\Builder.php(303): Illuminate\Database\Eloquent\Builder->getModels(Array)
#8 C:\wamp\www\lumen\vendor\illuminate\database\Eloquent\Model.php(645): Illuminate\Database\Eloquent\Builder->get(Array)
#9 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(39): Illuminate\Database\Eloquent\Model::all('Domlur')
#10 [internal function]: App\Http\Controllers\PlaceController->getPlace(Object(Illuminate\Http\Request))
#11 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'getPlace', Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#18 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#19 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#20 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#21 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#22 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42S22]: Column not found: 1054 Unknown column 'Domlur' in 'field list' (SQL: select `Domlur` from `places`)' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:669
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('select `Domlur`...', Array, Object(Closure))
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(342): Illuminate\Database\Connection->run('select `Domlur`...', Array, Object(Closure))
#2 C:\wamp\www\lumen\vendor\illuminate\database\Query\Builder.php(1461): Illuminate\Database\Connection->select('select `Domlur`...', Array, true)
#3 C:\wamp\www\lumen\vendor\illuminate\database\Query\Builder.php(1447): Illuminate\Database\Query\Builder->runSelect()
#4 C:\wamp\www\lumen\vendor\illuminate\database\Eloquent\Builder.php(569): Illuminate\Database\Query\Builder->get(Array)
#5 C:\wamp\www\lumen\vendor\illuminate\database\Eloquent\Builder.php(303): Illuminate\Database\Eloquent\Builder->getModels(Array)
#6 C:\wamp\www\lumen\vendor\illuminate\database\Eloquent\Model.php(645): Illuminate\Database\Eloquent\Builder->get(Array)
#7 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(39): Illuminate\Database\Eloquent\Model::all('Domlur')
#8 [internal function]: App\Http\Controllers\PlaceController->getPlace(Object(Illuminate\Http\Request))
#9 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'getPlace', Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#18 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#19 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#20 {main}  
[2016-03-10 06:33:31] lumen.ERROR: exception 'ErrorException' with message 'Undefined variable: tabel' in C:\wamp\www\lumen\database\migrations\2016_03_10_062835_create_users_table.php:18
Stack trace:
#0 C:\wamp\www\lumen\database\migrations\2016_03_10_062835_create_users_table.php(18): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined varia...', 'C:\\wamp\\www\\lum...', 18, Array)
#1 C:\wamp\www\lumen\vendor\illuminate\database\Schema\Builder.php(144): CreateUsersTable->{closure}(Object(Illuminate\Database\Schema\Blueprint))
#2 C:\wamp\www\lumen\vendor\illuminate\support\Facades\Facade.php(221): Illuminate\Database\Schema\Builder->create('users', Object(Closure))
#3 C:\wamp\www\lumen\database\migrations\2016_03_10_062835_create_users_table.php(24): Illuminate\Support\Facades\Facade::__callStatic('create', Array)
#4 C:\wamp\www\lumen\database\migrations\2016_03_10_062835_create_users_table.php(24): Illuminate\Support\Facades\Schema::create('users', Object(Closure))
#5 C:\wamp\www\lumen\vendor\illuminate\database\Migrations\Migrator.php(147): CreateUsersTable->up()
#6 C:\wamp\www\lumen\vendor\illuminate\database\Migrations\Migrator.php(117): Illuminate\Database\Migrations\Migrator->runUp('2016_03_10_0628...', 2, false)
#7 C:\wamp\www\lumen\vendor\illuminate\database\Migrations\Migrator.php(86): Illuminate\Database\Migrations\Migrator->runMigrationList(Array, Array)
#8 C:\wamp\www\lumen\vendor\illuminate\database\Console\Migrations\MigrateCommand.php(77): Illuminate\Database\Migrations\Migrator->run('C:\\wamp\\www\\lum...', Array)
#9 [internal function]: Illuminate\Database\Console\Migrations\MigrateCommand->fire()
#10 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#11 C:\wamp\www\lumen\vendor\illuminate\console\Command.php(169): Illuminate\Container\Container->call(Array)
#12 C:\wamp\www\lumen\vendor\symfony\console\Command\Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#13 C:\wamp\www\lumen\vendor\illuminate\console\Command.php(155): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#14 C:\wamp\www\lumen\vendor\symfony\console\Application.php(791): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#15 C:\wamp\www\lumen\vendor\symfony\console\Application.php(186): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Migrations\MigrateCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 C:\wamp\www\lumen\vendor\symfony\console\Application.php(117): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Console\Kernel.php(69): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 C:\wamp\www\lumen\artisan(35): Laravel\Lumen\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 {main}  
[2016-03-10 06:47:50] lumen.ERROR: exception 'ErrorException' with message 'Undefined variable: request' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:81
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(81): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined varia...', 'C:\\wamp\\www\\lum...', 81, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->register()
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'register', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 07:16:16] lumen.ERROR: exception 'ErrorException' with message 'Undefined variable: plac' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:44
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(44): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined varia...', 'C:\\wamp\\www\\lum...', 44, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->getPlace(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'getPlace', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:23:01] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:25:10] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:26:05] lumen.ERROR: exception 'ErrorException' with message 'Trying to get property of non-object' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:84
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(84): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Trying to get p...', 'C:\\wamp\\www\\lum...', 84, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login()
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:30:26] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:37:16] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function num_rows() on a non-object' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:84
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-10 09:41:57] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:42:32] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(true)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(true)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(true)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(true)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 09:43:06] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(true)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(true)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(true)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(true)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 10:04:50] lumen.ERROR: exception 'ErrorException' with message 'Undefined offset: 0' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:84
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(84): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined offse...', 'C:\\wamp\\www\\lum...', 84, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login()
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 10:12:26] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 11:31:40] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 11:34:56] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 11:35:21] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 11:36:32] lumen.ERROR: exception 'ErrorException' with message 'Use of undefined constant console - assumed 'console'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:86
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(86): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Use of undefine...', 'C:\\wamp\\www\\lum...', 86, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 11:58:41] lumen.ERROR: exception 'ErrorException' with message 'Undefined variable: results' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:87
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(87): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Undefined varia...', 'C:\\wamp\\www\\lum...', 87, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 17:18:39] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 17:19:02] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-10 17:21:08] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 05:10:00] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:10:25] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:12:14] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:35:55] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:39:54] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:40:39] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:40:47] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:43:36] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:44:18] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:44:38] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:45:03] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:45:23] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:48:58] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:56:34] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 05:57:46] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:01:26] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:01:48] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot redeclare App\Http\Controllers\PlaceController::login()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:94
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:04:03] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:04:29] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:04:54] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:06:32] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:14:05] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:14:53] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:15:14] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:15:36] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:16:00] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 06:16:07] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected '}'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:97
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 10:04:16] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 11:24:38] lumen.ERROR: exception 'ErrorException' with message 'Use of undefined constant length - assumed 'length'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:98
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(98): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Use of undefine...', 'C:\\wamp\\www\\lum...', 98, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 11:24:48] lumen.ERROR: exception 'ErrorException' with message 'Use of undefined constant length - assumed 'length'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:98
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(98): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Use of undefine...', 'C:\\wamp\\www\\lum...', 98, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 11:25:07] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Cannot use [] for reading' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:98
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-11 11:26:44] lumen.ERROR: exception 'ErrorException' with message 'Use of undefined constant length - assumed 'length'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:98
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(98): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Use of undefine...', 'C:\\wamp\\www\\lum...', 98, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->login(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 11:30:37] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(false)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(false)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(false)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(false)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 11:36:10] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(true)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(true)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(true)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(true)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 12:27:07] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(true)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(true)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(true)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(true)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-11 12:27:29] lumen.ERROR: exception 'UnexpectedValueException' with message 'The Response content must be a string or object implementing __toString(), "boolean" given.' in C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php:397
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\http\Response.php(56): Symfony\Component\HttpFoundation\Response->setContent(true)
#1 C:\wamp\www\lumen\vendor\symfony\http-foundation\Response.php(199): Illuminate\Http\Response->setContent(true)
#2 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(638): Symfony\Component\HttpFoundation\Response->__construct(true)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(582): Laravel\Lumen\Application->prepareResponse(true)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'login', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
[2016-03-15 09:11:17] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','','') VALUES('',,'','','squa' at line 1' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:408
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(408): PDO->prepare('INSERT INTO pla...')
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(662): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'INSERT INTO pla...', Array)
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#3 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#4 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#5 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#6 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#7 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#8 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#9 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#10 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#21 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','','') VALUES('',,'','','squa' at line 1 (SQL: INSERT INTO places ('id','place','pincode','bed','square_feet','price','','') VALUES('',,'','','square_feet','price','',''))' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:669
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#3 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#4 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#5 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#6 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#7 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#8 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#18 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#19 {main}  
[2016-03-15 09:12:10] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','created_at','updated_at') VA' at line 1' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:408
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(408): PDO->prepare('INSERT INTO pla...')
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(662): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'INSERT INTO pla...', Array)
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#3 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#4 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#5 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#6 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#7 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#8 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#9 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#10 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#21 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','created_at','updated_at') VA' at line 1 (SQL: INSERT INTO places ('id','place','pincode','bed','square_feet','price','created_at','updated_at') VALUES('',,'','','square_feet','price','',''))' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:669
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#3 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#4 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#5 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#6 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#7 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#8 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#18 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#19 {main}  
[2016-03-15 09:13:03] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','created_at','updated_at') VA' at line 1' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:408
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(408): PDO->prepare('INSERT INTO pla...')
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(662): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'INSERT INTO pla...', Array)
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#3 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#4 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#5 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#6 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#7 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#8 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#9 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#10 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#21 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','created_at','updated_at') VA' at line 1 (SQL: INSERT INTO places ('id','place','pincode','bed','square_feet','price','created_at','updated_at') VALUES('',,'','','square_feet','price','',''))' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:669
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#3 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#4 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#5 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#6 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#7 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#8 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#18 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#19 {main}  
[2016-03-15 09:15:13] lumen.ERROR: exception 'PDOException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','created_at','updated_at') VA' at line 1' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:408
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(408): PDO->prepare('INSERT INTO pla...')
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(662): Illuminate\Database\Connection->Illuminate\Database\{closure}(Object(Illuminate\Database\MySqlConnection), 'INSERT INTO pla...', Array)
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#3 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#4 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#5 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#6 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#7 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#8 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#9 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#10 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#18 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#19 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#20 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#21 {main}

Next exception 'Illuminate\Database\QueryException' with message 'SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''id','place','pincode','bed','square_feet','price','created_at','updated_at') VA' at line 1 (SQL: INSERT INTO places ('id','place','pincode','bed','square_feet','price','created_at','updated_at') VALUES('',,'','','square_feet','price','now()','now()'))' in C:\wamp\www\lumen\vendor\illuminate\database\Connection.php:669
Stack trace:
#0 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(629): Illuminate\Database\Connection->runQueryCallback('INSERT INTO pla...', Array, Object(Closure))
#1 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(409): Illuminate\Database\Connection->run('INSERT INTO pla...', Array, Object(Closure))
#2 C:\wamp\www\lumen\vendor\illuminate\database\Connection.php(365): Illuminate\Database\Connection->statement('INSERT INTO pla...', Array)
#3 [internal function]: Illuminate\Database\Connection->insert('INSERT INTO pla...')
#4 C:\wamp\www\lumen\vendor\illuminate\database\DatabaseManager.php(317): call_user_func_array(Array, Array)
#5 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->__call('insert', Array)
#6 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(54): Illuminate\Database\DatabaseManager->insert('INSERT INTO pla...')
#7 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#8 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#12 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#13 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#14 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#15 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#16 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#17 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#18 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#19 {main}  
[2016-03-17 07:02:03] lumen.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to undefined function App\Http\Controllers\alert()' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:61
Stack trace:
#0 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RegistersExceptionHandlers.php(55): Laravel\Lumen\Application->handleShutdown()
#1 [internal function]: Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#2 {main}  
[2016-03-17 07:02:44] lumen.ERROR: exception 'ErrorException' with message 'Use of undefined constant console - assumed 'console'' in C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php:61
Stack trace:
#0 C:\wamp\www\lumen\app\Http\Controllers\PlaceController.php(61): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}(8, 'Use of undefine...', 'C:\\wamp\\www\\lum...', 61, Array)
#1 [internal function]: App\Http\Controllers\PlaceController->savePlace(Object(Illuminate\Http\Request))
#2 C:\wamp\www\lumen\vendor\illuminate\container\Container.php(507): call_user_func_array(Array, Array)
#3 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(581): Illuminate\Container\Container->call(Array, Array)
#4 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(548): Laravel\Lumen\Application->callControllerCallable(Array, Array)
#5 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(521): Laravel\Lumen\Application->callLumenController(Object(App\Http\Controllers\PlaceController), 'savePlace', Array)
#6 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(489): Laravel\Lumen\Application->callControllerAction(Array)
#7 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(474): Laravel\Lumen\Application->callActionOnArrayBasedRoute(Array)
#8 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(376): Laravel\Lumen\Application->handleFoundRoute(Array)
#9 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(624): Laravel\Lumen\Application->Laravel\Lumen\Concerns\{closure}()
#10 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(382): Laravel\Lumen\Application->sendThroughPipeline(Array, Object(Closure))
#11 C:\wamp\www\lumen\vendor\laravel\lumen-framework\src\Concerns\RoutesRequests.php(327): Laravel\Lumen\Application->dispatch(NULL)
#12 C:\wamp\www\lumen\public\index.php(28): Laravel\Lumen\Application->run()
#13 {main}  
